omod/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_9/ConceptController1_9Test.java

```diff
@@ -71,7 +71,7 @@ public void shouldFindNumericConceptsByQueryString() throws Exception {
 		        "full"))));
 		List<Object> results = Util.getResultsList(response);
 		
-		Assert.assertEquals(2, results.size());
+		Assert.assertEquals(1, results.size());
```

Comment:
This was changed from 1 to 2 when upgrading to build against 2.7.x.  "1" appears to be the correct answer, not sure why it was "2" when building against 2.8.x, but back to what I expect now.